config/servicemappings/cloudbuild.yaml

```diff
@@ -22,6 +22,10 @@ spec:
   version: v1beta1
   serviceHostName: "cloudbuild.googleapis.com"
   resources:
+    - name: google_cloudbuild_workerpool
```

Comment:
You've probably have broken most of the service mapping unit tests so you'll need to fix them.

Reply:
Fixed. This is better than I expected 😄